packages/extensionmanager/src/model.ts

```diff
@@ -343,14 +353,20 @@ export class ListModel extends VDomModel {
    * Install an extension.
    *
    * @param entry An entry indicating which extension to install.
-   */
-  async install(entry: IEntry): Promise<void> {
-    await this.performAction('install', entry).then(data => {
-      if (data.status !== 'ok') {
-        reportInstallError(entry.name, data.message, this.translator);
+   * @param useLatestVersion Whether to use the latest version to install.
+   */
+  async install(
+    entry: IEntry,
+    useLatestVersion: boolean = false
```

Comment:
We prefer to use object that are more flexible to extend an API in a non-breaking way.

```suggestion
    options: { useVersion?: string }
```

> e.g. in the future we may want to add an option `force` to the install action.